src/main/java/Deadline.java

```diff
@@ -0,0 +1,12 @@
+public class Deadline extends Task {
+    String by;
+    public Deadline(String desc, String by) throws Exception{
```

Comment:
```suggestion
    String by;
    
    public Deadline(String desc, String by) throws Exception{
```

Reply:
In addition to this change, don't forget to add white space in the method definition.
```suggestion
    String by;
    
    public Deadline(String desc, String by) throws Exception {
```